go.mod

```diff
@@ -4,11 +4,10 @@ go 1.19
 
 require (
 	github.com/onsi/gomega v1.27.10
-	github.com/openshift-online/ocm-sdk-go v0.1.368
 	github.com/openshift/api v0.0.0-20230213134911-7ba313770556
-	github.com/openshift/client-go v0.0.0-20221019143426-16aed247da5c
-	github.com/project-codeflare/instascale v0.2.1
-	github.com/project-codeflare/multi-cluster-app-dispatcher v1.37.1
+	github.com/project-codeflare/codeflare-common v0.0.0-20231023092720-93d03492db16
+	github.com/project-codeflare/instascale v0.2.0
+	github.com/project-codeflare/multi-cluster-app-dispatcher v1.37.0
```

Comment:
Looks like unintentional change, can you revert this?

Reply:
Otherwise the PR looks fine

Reply:
Possibly a mistake when rebasing, thank you. Updated now.